src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -0,0 +1,587 @@
+
+#include <opm/parser/eclipse/EclipseState/EclipseState.hpp>
+#include <opm/parser/eclipse/EclipseState/Runspec.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp>
+
+
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQConfig.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQActive.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQDefine.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQAssign.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/Action/ActionX.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQEnums.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQParams.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQFunctionTable.hpp>
+
+#include <algorithm>
+#include <cstddef>
+#include <cstring>
+#include <string>
+#include <iostream>
+
+// #####################################################################
+// Class Opm::RestartIO::Helpers::AggregateGroupData
+// ---------------------------------------------------------------------
+
+
+namespace {
+    const std::map<std::string, int> lhsQuantityToIndex = {
+                                                           {"F",   1},
+                                                           {"W",   2},
+                                                           {"G",   3},
+                                                           {"D",  10},
+                                                           {"M",  11},
+                                                           {"Y",  12},
+    }; 
+
+    const std::map<std::string, double> monthToNo = {
+                                                           {"JAN",   1.},
+                                                           {"FEB",   2.},
+                                                           {"MAR",   3.},
+                                                           {"APR",   4.},
+                                                           {"MAY",   5.},
+                                                           {"JUN",   6.},
+                                                           {"JUL",   7.},
+                                                           {"AUG",   8.},
+                                                           {"SEP",   9.},
+                                                           {"OCT",  10.},
+                                                           {"NOV",  11.},
+                                                           {"DEC",  12.},
+    }; 
+
+    
+            const std::map<std::string, int> rhsQuantityToIndex = {
+                                                           {"F",   1},
+                                                           {"W",   2},
+                                                           {"G",   3},
+        }; 
+        
+        using logic_enum = Opm::Action::Condition::Logical;
+        const std::map<logic_enum, int> logicalToIndex_13 = {
```

Comment:
This is nitpicking - but it is the type of error which has led to subtle errors in the past - at least I have made them: "The integer values 0,1,2 which you convert to here are *not* indices - they are not used to look up in some vector; they are just integer ID's"; so I would prefer if you called it e.g. `logicalToInt_13`.